lighthouse/tests/beacon_node.rs

```diff
@@ -2692,3 +2693,13 @@ fn genesis_state_url_value() {
             assert_eq!(config.genesis_state_url_timeout, Duration::from_secs(42));
         });
 }
+
+#[test]
+fn beacon_node_backend_override() {
+    CommandLineTest::new()
+        .flag("beacon-node-backend", Some("leveldb"))
```

Comment:
It would be ideal to have a test that sets this to `redb`, as the flag was actually completely defunct and this test still passed. But I think this would require compiling with `beacon-node-redb` as a default feature. Hmm :thinking: